app.js

```diff
@@ -149,8 +129,13 @@ async function recoveryMethodsFor(account) {
     if (!account) return [];
 
     return await account.getRecoveryMethods({
-        attributes: ['createdAt', 'detail', 'kind', 'publicKey']
-    }).map(m => m.toJSON());
+        attributes: ['createdAt', 'detail', 'kind', 'publicKey', 'securityCode']
+    }).map(method => {
+        const json = method.toJSON();
+        json.confirmed = !method.securityCode;
```

Comment:
best to define virtual field for this https://sequelize.org/master/manual/getters-setters-virtuals.html#virtual-fields